docs/config/node.rst

```diff
@@ -556,13 +556,15 @@ Layer Security (TLS).
 .. _ssl.transport.mode:
 
 **ssl.transport.mode**
-  | *Default:* ``off``
+  | *Default:* ``legacy``
   | *Runtime:* ``no``
 
   For communication between nodes, choose:
 
   ``off``
     SSL cannot be used
+  ``legacy``
+    SSL cannot be used and HBA will not be used even if HBA is enabled
```

Comment:
```suggestion
    SSL is not used. If HBA is enabled, transport connections won't be verified. Any reachable host can establish a connection.
```

To clarify that this doesn't disable HBA entirely, but that it only affects how transport connections are treated.